.github/issue_template.md

```diff
@@ -0,0 +1,31 @@
+## Expected Behavior
+
+...
+
+## Actual Behavior
+
+...
+
+## Steps to Reproduce the Problem
+
+  1. ...
+  1. ...
+  1. ...
+  
+  
+## Logs, error output, etc.
+ 
+ (`epoch.yaml` configuration file, `log/epoch.log` log file plus other related logs, e.g., `log/epoch_mining.log`; if it’s long, please paste to https://ghostbin.com/ and insert the link here)
+ 
+ ...
+
+
+## Specifications
+
+
+  - Virtualization: [*none / metal / container / VM*]
+  - Hardware specs: 
+    * RAM: 
+    * CPU:
+  - OS: [*distribution, version*]
+  - Node Version:
```

Comment:
we need to know how the OTP was installed in the system (brew/source/packages) and which OTP is it

Also we need to know how `epoch` was installed (source/packages)

Reply:
@sennui I think we should "support" only the official release deployments, our master is not stable enough to support deploys installed from it, is it?

If we agree to support only package installs from official release, then we ship with OTP already. The only external dependancy AFAIK is libssl.